src/Tag/Controller/Element/AssignController.php

```diff
@@ -48,10 +48,10 @@ public function __construct(
     /**
      * @throws ElementSavingFailedException|NotFoundException
      */
-    #[Route('/tags/{elementType}/{id}/{tagId}', name: 'pimcore_studio_api_assign_element_tag', methods: ['POST'])]
+    #[Route('/tags/assign/{elementType}/{id}/{tagId}', name: 'pimcore_studio_api_assign_element_tag', methods: ['POST'])]
```

Comment:
Sonar is now complaining that the line is too long :D

Pls split it up in multiple lines..